invenio_records_lom/records/api.py

```diff
@@ -0,0 +1,134 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2021 Graz University of Technology.
+#
+# invenio-records-lom is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Classes for "API"-objects, whose fields represent database-information.
+
+Use flask.current_app['invenio-records-lom'].records_service to interact with.
+"""
+
+# fields execute actions on creating, editing, publishing, ... the object
```

Comment:
is this comment for documentation? should it be moved up to the module doc-string?